pkg/systemd/reporting.jsx

```diff
@@ -116,114 +104,139 @@ class FAFWorkflowRow extends React.Component {
                     }
 
                     if (reported) {
-                        this.setState({
-                            problemState: ProblemState.REPORTED,
-                            reportLinks,
-                        });
+                        setProblemState(ProblemState.REPORTED);
+                        setReportLinks(reportLinks);
                     }
                 });
-    }
+    };
 
-    _onCancelButtonClick(event) {
-        this.state.process.close("canceled");
-    }
+    useInit(() => {
+        updateStatusFromBus();
+    });
 
-    _onReportButtonClick(event) {
-        this.setState({ problemState: ProblemState.UNREPORTABLE });
+    const onCancelButtonClick = _event => process.close("canceled");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19403-20231002-082424-b39d5ff6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)